src/k8s-engine/minikube.js

```diff
@@ -74,6 +74,7 @@ class Minikube extends EventEmitter {
       let opts = {};
       opts.env = { ...process.env };
       opts.env['MINIKUBE_HOME'] = paths.data();
+      opts.env['PATH'] = resources.get(os.platform()) + ((opts.env['PATH'] === '') ? '' : ':') + opts.env['PATH'];
```

Comment:
```suggestion
      let resourcePath = resources.get(os.platform();
      let path = Array.from(opts.env.PATH?.split(path.delimiter) ?? []);
      opts.env.PATH = path.unshift(resourcePath).join(path.delimiter);
```
That is:
- `opts.env['PATH']` would be `undefined` rather than empty string if it's not set, right?
- `path.delimiter` is the separator, which also works on Windows. We don't need it _yet_ of course.

Reply:
That's a more elegant solution than mine. I had to fix a couple syntax errors, copy it to a couple places, and add the path import.